doc/reference/cmdline.rst

```diff
@@ -464,7 +464,7 @@ Configuration file
 .. program:: odoo-bin
 
 Most of the command-line options can also be specified via a configuration
-file. Most of the time, they use similar names with the prefix ``-`` removed
+file within the section ``[options]``. Most of the time, they use similar names with the prefix ``-`` removed
```

Comment:
fil-paragraph

Reply:
Also the bit about the `options` section should maybe go right above the example snippet, after we tell people where the file itself lives (the more "concrete" section, if you will), I find the placement here to be a bit odd and the reading to not really flow well.

Reply:
I took your comment into account, how do you like it now ?